### PR TITLE
ci: fix missing say command

### PR DESCRIPTION
**Pull request by rb-determined-ai**

### CODE REVIEW

File: .github/workflows/lint-secrets.yml [hunk 1]
```diff
@@ -31,6 +31,8 @@ jobs:
           rm -rf .tmp
       - name: Configure git-secrets
         run: |
+          # workaround git-secrets requiring the say command
+          sudo ln -s "$(which echo)" /usr/local/bin/say
```
Comment (dannysauer) [inline .github/workflows/lint-secrets.yml:35]: The action already runs as root. :)
```suggestion
          ln -s "$(which echo)" /usr/local/bin/say
```

Comment (rb-determined-ai) [reply .github/workflows/lint-secrets.yml]: you've never heard of super-root?

**Review by dannysauer — APPROVED** [review]
This is fine - but probably oughtta do the same thing for the identical check in .circleci/config

**Comment by dannysauer** [comment]
Ok; I won't ask how this is continuing to work without the same workaround in the circleci environment. #SeeNoEvil

**Comment by sparr** [comment]
https://github.com/awslabs/git-secrets/pull/221 fixes this upstream and has been merged. You should be able to remove your workaround.

**Pull request merged**